website_sale_wishlist/demo/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+<template id="assets_frontend_demo" inherit_id="website.assets_frontend">
```

Comment:
Not in this case, because there is already an `assets_frontend` view in this addon, but this one is just for demo environments, to run tests.